src/BlockTypes/StoreNotices.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+
+/**
+ * StoreNotices class.
+ */
+class StoreNotices extends AbstractBlock {
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'store-notices';
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content Block content.
+	 * @param WP_Block $block Block instance.
+	 *
+	 * @return string | void Rendered block output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		ob_start();
+		woocommerce_output_all_notices();
+		$notices = ob_get_clean();
+
+		if ( ! $notices ) {
+			return;
+		}
+
+		$classname          = isset( $attributes['className'] ) ? $attributes['className'] : '';
+		$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes );
+
+		return "<div class='woocommerce wc-block-store-notices " . esc_attr( $classes_and_styles['classes'] ) . ' ' . esc_attr( $classname ) . "' style='" . esc_attr( $classes_and_styles['styles'] ) . "'>" . wc_kses_notice( $notices ) . '</div>';
```

Comment:
[Option] Just my preference, for the return string that contains two or more variables, I usually use `sprintf` to make the code easier to read. What do you think?

Reply:
Same for the [Catalog Sorting](https://github.com/woocommerce/woocommerce-blocks/pull/8122/files#diff-f91154f1d10ad36297ebdef901244f8a838c0f2173767ed388d878f33bb19b3cR40) block.

Reply:
Agree!